oauth2client/client.py

```diff
@@ -1617,6 +1617,18 @@ def _revoke(self, http_request):
         """
         self._do_revoke(http_request, self.access_token)
 
+    def sign_blob(self, blob):
+        """Cryptographically sign a blob (of bytes).
+
+        Args:
+            blob: bytes, Message to be signed.
+
+        Returns:
+            tuple, A pair of the private key ID used to sign the blob and
+            the signed contents.
+        """
+        raise NotImplementedError
```

Comment:
I prefer the explicit `()` in `raise NotImplementedError()`; humor me if you don't care either way?